src/Domain/Configuration.php

```diff
@@ -160,6 +185,46 @@ public function getPreset(): string
         return $this->preset;
     }
 
+    /**
+     * @return float
+     */
+    public function getMinQuality(): float
+    {
+        return (float) ($this->requirements['min-quality'] ?? 0);
+    }
+
+    /**
+     * @return float
+     */
+    public function getMinComplexity(): float
+    {
+        return (float) ($this->requirements['min-complexity'] ?? 0);
+    }
+
+    /**
+     * @return float
+     */
+    public function getMinArchitecture(): float
+    {
+        return (float) ($this->requirements['min-architecture'] ?? 0);
+    }
+
+    /**
+     * @return float
+     */
+    public function getMinStyle(): float
+    {
+        return (float) ($this->requirements['min-style'] ?? 0);
+    }
+
+    /**
+     * @return bool
+     */
+    public function getDisableSecurityCheck(): bool
```

Comment:
Perhaps we should rename this in `isSecurityCheckDisabled()` ? (not blocking anyway)

Reply:
sure sounds great :)